kontrakter/vegtrafikkulykke/trafikkulykke/1.0/trafikkulykke.schema.json

```diff
@@ -652,7 +648,12 @@
           "description": "Informasjon om person involvert i ulykken."
         }
       },
-      "required": ["id"],
+      "required": [
+        "enhetsbokstav",
+        "registreringsnummer",
```

Comment:
Registreringsnummer kan være null i flere tilfeller, eks. ved enhetstype=gående :)

Reply:
Ah, der var jeg for rask ja - tittet bare på påkrevde felter i BL her, litt misvisende 😅